infra/fendermint/scripts/subnet.toml

```diff
@@ -122,7 +122,7 @@ script.pre = "mkdir -p ${BASE_DIR}/${NODE_NAME}/${KEYS_SUBDIR}; cp ${PRIVATE_KEY
 
 [tasks.subnet-fetch-genesis]
 extend = "fendermint-tool"
-env = { "CMD" = "genesis --genesis-file /data/genesis.json ipc from-parent --subnet-id ${SUBNET_ID} -p ${PARENT_ENDPOINT}  --parent-gateway ${PARENT_GATEWAY}  --parent-registry ${PARENT_REGISTRY} --base-fee ${BASE_FEE} --power-scale ${POWER_SCALE}" }
+env = { "CMD" = "genesis --genesis-file /data/genesis.json ipc from-parent --subnet-id ${SUBNET_ID} --parent-network-type ${PARENT_NETWORK_TYPE} -p ${PARENT_ENDPOINT}  --parent-gateway ${PARENT_GATEWAY}  --parent-registry ${PARENT_REGISTRY} --base-fee ${BASE_FEE} --power-scale ${POWER_SCALE}" }
```

Comment:
hey @OrestisAlpos after looking a bit into this part of IPC, I think the `PARENT_NETWORK_TYPE` should be deduced from the `SUBNET_ID` – so maybe this needs to be tackled beforehand